ebpf-shared/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 #[repr(C)]
 #[derive(Clone, Copy, Debug, PartialEq, Eq)]
 pub struct Signal {
+    pub cgroupid: u64,
```

Comment:
`cgroup_id`?

Reply:
Sorry for some reason I always go minimal when writing C :)